pages/tech-of-the-month/VoteSystem.vue

```diff
@@ -0,0 +1,256 @@
+<script>
+import RadioLayout from './RadioLayout.vue'
+import { addVote, checkUser, login } from '~/assets/database/firebase.js'
+export default {
+  components: {
+    apexcharts: () => import('vue-apexcharts'),
+    RadioLayout
+  },
+  props: {
+    databaseSeries: {
+      type: Array,
+      default: () => [0, 0, 0]
+    },
+    databaseLabels: {
+      type: Array,
+      default: () => []
+    },
+    month: {
+      type: String,
+      default: ''
+    },
+    year: {
+      type: String,
+      default: ''
+    }
+  },
+  data () {
+    return {
+      title: ['First Choice', 'Second Choice', 'Third Choice'],
```

Comment:
Make the choice columns dynamic rather than hardcoded. Once updating Firebase, the appropriate number of choices & columns should be displayed